app/editor/src/features/admin/event-of-the-day/styled/EventOfTheDayList.tsx

```diff
@@ -95,4 +95,15 @@ export const EventOfTheDayList = styled.div`
     transition: all 1s;
     transform: rotateY(180deg);
   }
+
+  .icon-refresh {
```

Comment:
You shouldn't need this if the new component `RefreshButton` has been styled.

Reply:
RefreshButton is only added to Subscriber. And there will be a separate story for Editor site.